projects/fsastorefrontlib/src/assets/translations/en/sales-illustration.en.ts

```diff
@@ -0,0 +1,17 @@
+export const salesIllustration = {
+  salesIllustration: {
+    expectedSavings: 'Expected Amount',
+    guaranteedSavings: 'Guaranteed Amount',
+    monthlyAnnuity: 'Monthly Annuity',
+    savingsDetails: 'Your Savings Details',
+    chartView: 'Chart View',
+    tableView: 'Table View',
+    year: 'Year',
+    interest: 'Interest',
+    itemsPerPage: 'Items Per Page',
+    productWith: 'Product with code',
+    notAvailable: 'is not available for sales illustration',
+    mainTitle: 'Savings - {{ cartDisplayName }} Sales Illustration',
+    savingsProgress: 'Your Savings Progress',
```

Comment:
Perhaps move this to quote.ts -> details -> insurances_savings because `salesIllustration` shouldn't be fixed to any product.